packages/components/modal/demo/Async.vue

```diff
@@ -1,27 +1,22 @@
 <template>
-  <IxSpace>
-    <IxButton @click="visible = !visible">Change visible</IxButton>
-    <IxButton mode="primary" @click="openModal">Open modal</IxButton>
-  </IxSpace>
-  <IxModal v-model:visible="visible" title="This is title" type="confirm" @ok="onOk">
-    <p>Some contents...</p>
+  <IxButton mode="primary" @click="showModal">Open Modal with async logic</IxButton>
+  <IxModal v-model:visible="visible" title="标题" type="confirm" :onOk="onOk">
+    <p>内容文本...</p>
```

Comment:
```suggestion
  <IxModal v-model:visible="visible" title="This is title" type="confirm" @ok="onOk">
    <p>Some contents..</p>
```
要么就全中文嘛，按钮是英文，弹窗是中文，好奇怪。 还有 `:onOk` => `@ok`